tests/integration/utils/kiali/kiali_client.go

```diff
@@ -311,6 +311,21 @@ func ServicesList(namespace string) (*ServiceListJson, error) {
 	}
 }
 
+func ClusterServicesList(namespace string) (*ServiceListJson, error) {
+	body, _, _, err := httpGETWithRetry(client.kialiURL+"/api/clusters/services?namespaces="+namespace, client.GetAuth(), TIMEOUT, nil, client.kialiCookies)
+	if err == nil {
+		serviceList := new(ServiceListJson)
+		err = json.Unmarshal(body, &serviceList)
+		if err == nil {
+			return serviceList, nil
+		} else {
+			return nil, err
+		}
+	} else {
+		return nil, err
+	}
+}
```

Comment:
You can use `getRequestAndUnmarshalInto(url, serviceList)` to save on typing a bit. It will also check the status code before unmarshaling the json so that the actual api error doesn't get swallowed by the unmarshal error when the response is not 200.